modules/apps/cookies/cookies-banner-web/src/main/resources/META-INF/resources/product_analytics_banner/js/ProductAnalyticsBanner.js

```diff
@@ -205,11 +208,19 @@ function isCookieTypesAccepted(cookieTypes) {
 }
 
 function setBannerVisibility(productAnalyticsBanner) {
+	const cookieBanner = document.querySelector('.cookies-banner');
```

Comment:
If the `.cookies-banner` isn't present, then an error will be thrown on line `215` or `218`, since `cookieBanner` is not defined. This can happen if the Cookie Manager isn't enabled, or if we've already accepted/declined the Cookie Manager prompt.

Reply:
I'll check and add an if for this, but I didn't found this error in my tests with firefox